src/main/java/PageObjects/CreateAccountPage.java

```diff
@@ -0,0 +1,72 @@
+package PageObjects;
+
+import DriverTools.SingleDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.io.IOException;
+
+public class CreateAccountPage {
+    private final WebDriver driver;
+    private final String phoneNumber = "123456";
+    private final String lastName = "LastName";
+    private final String firstName = "FirstName";
+    private final String password = "passwd";
+    private final String address = "Minsk, Chapaeva, 5";
+    private final String city = "Minsk";
+    private final String postCode = "12345";
```

Comment:
I recommend to add "default" in names, f.e. defaultPhoneNumber, because it looks like test data right now. And test data must be in test classes. As far as these values are just defaults, then can be here.